src/Core/Silk.NET.BuildTools/Bind/ProjectWriter.cs

```diff
@@ -60,21 +60,33 @@ public static void Write(this Project project, string folder, Profile profile, B
 
             project.WriteProjectFile(folder, profile, task);
 
+            Project coreProject = profile.Projects["Core"];
+
             project.Structs.ForEach
             (
-                x => x.WriteStruct
-                (
-                    GetFileName(x.Name, ".gen.cs", folder, ProfileWriter.StructsSubfolder), profile, project, task
-                )
-            );
+                x =>
+                {
+                    if(coreProject != project && coreProject.Structs.Any(y => y.NativeName == x.NativeName))
+                        return;
+
+                    x.WriteStruct
+                                    (
+                                        GetFileName(x.Name, ".gen.cs", folder, ProfileWriter.StructsSubfolder), profile, project, task
+                                    );
+                });
```

Comment:
Formatting is a bit weird here (everywhere in file)

We use allman braces i.e. the level of indentation/formatting you'd expect for a curly bracket is exactly the same for regular brackets.

Reply:
I followed the formatting rules that rider picked up on, eg. the `.editorconfig`